atom/browser/api/atom_api_window.cc

```diff
@@ -901,6 +909,8 @@ void Window::BuildPrototype(v8::Isolate* isolate,
                  &Window::SetVisibleOnAllWorkspaces)
       .SetMethod("isVisibleOnAllWorkspaces",
                  &Window::IsVisibleOnAllWorkspaces)
+      .SetMethod("setVibrancy", &Window::SetVibrancy)
+      .SetMethod("removeVibrancy", &Window::RemoveVibrancy)
```

Comment:
Looks like other methods in `BrowserWindow` like `setMenu`, `setParentWindow`, `setThumbnailClip` support clearing things via the setter, instead of a second `remove*` method.

For consistency it might be good to follow that pattern here, maybe `setVibrancy` with `null` removes it?